### PR TITLE
allow skipping of migrations via env var

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Run [Django](https://www.djangoproject.com) projects from source using [Gunicorn
 #### Step 1: Get your Django project in shape
 There are a few ways that your Django project needs to be set up in order to be compatible with this Docker image.
 
-**setup.py**  
+**setup.py**
```
Comment (JayH5) [inline README.md]: These double-spaces are newlines in markdown.

File: django-entrypoint.sh [hunk 2]
```diff
@@ -24,7 +24,9 @@ if [ "$1" = 'gunicorn' ]; then
   # Ultimately, the user shouldn't really be using a local DB and it's difficult
   # to offer support for all the cases in which a local DB might be created --
   # but here we do the minimum.
-  su-exec django django-admin migrate --noinput
+  if [ ! -n "$SKIP_MIGRATIONS" ]; then
```
Comment (JayH5) [inline django-entrypoint.sh]: `-z` is the opposite of `-n`

**Review by JayH5 — COMMENTED** [review]
We do tests for these kind of things now. Something along the lines of: https://github.com/praekeltfoundation/docker-django-bootstrap/blob/develop/example/test.py#L178

File: README.md (hunk 1 above)
Comment (smn) [reply README.md]: gah my editor seems to remove them automatically

**Review by JayH5 — APPROVED** [review]
:+1: thank you! 🍰

**Pull request merged**